eventheader/src/_internal.rs

```diff
@@ -5,14 +5,19 @@
 //! Internal implementation details for eventheader macros and eventheader_dynamic.
 //! Contents subject to change without notice.
 
+use core::mem;
+use core::ptr;
 use core::time::Duration;
 
+use crate::enums::ExtensionKind;
+
+pub use tracepoint::EventDataDescriptor;
+pub use tracepoint::TracepointState;
```

Comment:
Note: EventDataDescriptor and TracepointState are now re-exported from the `tracepoint` crate, no longer defined within this crate.